extensions-core/avro-extensions/src/main/java/io/druid/data/input/avro/AvroBytesDecoder.java

```diff
@@ -26,7 +26,8 @@
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type", defaultImpl = SchemaRepoBasedAvroBytesDecoder.class)
 @JsonSubTypes(value = {
-    @JsonSubTypes.Type(name = "schema_inline", value = SimpleAvroBytesDecoder.class),
+    @JsonSubTypes.Type(name = "schema_inline", value = InlineSchemaAvroBytesDecoder.class),
+    @JsonSubTypes.Type(name = "schemas_inline", value = InlineSchemasAvroBytesDecoder.class),
```

Comment:
This would be a really easy typo to miss. How would you feel about making the names a little more different, like `multiple_schemas_inline` or something?

I don't feel strongly enough about it to block if author desires to keep as is.